go/vt/vttablet/tabletserver/messager/cache.go

```diff
@@ -67,17 +71,24 @@ func (mh *messageHeap) Pop() interface{} {
 
 // cache is the cache for the messager.
```

Comment:
Add a comment here to explain the life cycle of the cache entries i.e. they are queued first, then popped because they will be sent out and then they are finally deleted once Discard() is called i.e. when there were sent out or when they were ACK'd.